motionplan/check.go

```diff
@@ -130,7 +116,24 @@ func checkPlanRelative(
 		return poseInWorld, nil
 	}
 
-	// Check out path pose is valid
+	plan := executionState.Plan()
+	wayPointIdx := executionState.Index()
+	startingInputs := executionState.currentInputs
+
+	currentInputs := executionState.CurrentInputs()
+	currentPoses := executionState.CurrentPoses()
+	if currentPoses == nil {
+		return errors.New("executionState had nil return from CurrentPoses")
+	}
+	localizationFrameName := checkFrame.Name() + "LocalizationFrame"
```

Comment:
This needs a better way to be passed in than the caller magically knowing that a thing named a certain way must exist.